Turn/include/Player.h

```diff
@@ -57,7 +61,8 @@ class Player : public Entity, public SoundMaker {
         int potions;
         int bombs;
         int weaponstrength;
-        int coins;
+        int coins=200;
+        int junk; //junk to be implimented in the future. 
```

Comment:
Usually pull requests only contain the relevant code for the one feature you want to merge. Since this isn't related to the Store feature I'd said remove this line for now